tutorials/getting_started/so_you_built_a_model.md

```diff
@@ -0,0 +1,340 @@
+and an input file (with one JSON input per line).
+
+Here we're using a custom model and predictor, so we'll also need to specify
+
+```
+--include-package my_library
+```
+
+just as we did during training, and
+
+```
+--predictor paper-classifier
+```
+
+so that AllenNLP knows which predictor to use.
+
+There are 10 papers in JSONL format at
+`tests/fixtures/s2_papers.jsonl`, so let's make some predictions on those:
+
+```bash
+python -m allennlp.run predict \
+    tests/fixtures/model.tar.gz \
+    tests/fixtures/s2_papers.jsonl \
+    --include-package my_library \
+    --predictor paper-classifier
+```
+
+When you run this it will print the ten test inputs and their predictions, each of which looks like:
+
+```
+prediction:  {"all_labels": ["AI", "ACL", "ML"], "logits": [0.008737504482269287, 0.22074833512306213, -0.005263201892375946], "class_probabilities": [0.31034138798713684, 0.38363200426101685, 0.3060266375541687], "label": "ACL"}
+```
+
+If you want your predictions to go to a file instead,
+you just need to specify an `--output-file`.
+
+## Running a Web Demo
+
+Once you have a trained model and a predictor,
+it's easy to run a simple web demo:
+
+```
+$ python -m allennlp.service.server_simple --help
+
+usage: server_simple.py [-h] [--archive-path ARCHIVE_PATH]
+                        [--predictor PREDICTOR] [--static-dir STATIC_DIR]
+                        [--title TITLE] [--field-name FIELD_NAME]
+                        [--include-package INCLUDE_PACKAGE]
+```
+
+Let's ignore the `--static-dir` flag for now and serve up the test fixture model:
+
+```
+python -m allennlp.service.server_simple \
+    --archive-path tests/fixtures/model.tar.gz \
+    --predictor paper-classifier \
+    --include-package my_library \
+    --title "Academic Paper Classifier" \
+    --field-name title \
+    --field-name paperAbstract
+```
+
+This will start a server at `localhost:8888` that serves up a really simple
+text-in-json-out version of your model:
+
+![demo screenshot](best_paper.png)
+
+Most of the options should be familiar to you by now.
+The two new ones are `--title`, which just specifies the title
+for the web demo, and `--field-name`, which specifies the input
+fields to create. The field names *must* exactly match the JSON keys
+that the predictor is expecting, or the demo won't work!
+
+## Customizing the Demo
+
+Our JSON "visualization" is not particularly impressive.
+We can customize it by putting our own `index.html`
+(and any other files we need) in some directory
+and calling `server_simple` with the `--static-dir` flag.
+In this case we don't need to provide a title or field names,
+as those will be implicitly provided by the HTML code.
+
+The simplest way to get started is to just "view source" on the demo
+and save the resulting file in some directory. I called my directory
+[`static_html`](https://github.com/allenai/allennlp-as-a-library-example/tree/master/static_html)
```

Comment:
Can we point to a tag on this repo, instead of master?

Reply:
should I tag what's there as v0.4.0?

Reply:
Yeah, sounds good to me.